docs/network-restrictions.md

```diff
@@ -0,0 +1,47 @@
+# Running behind a proxy
+
+If nodes only have access via a HTTP proxy, you can let the machine-controller configure all new nodes to use this proxy.
+For this the following flag must be set on the machine-controller side:
+```bash
+-node-http-proxy="http://192.168.1.1:3128"
+```
+This will set the following environment variables via /etc/environment on all nodes (lower & uppercase):
+- `HTTP_PROXY`
+- `HTTPS_PROXY`
+
+`NO_PROXY` can be configured using a dedicated flag:
+```bash
+-node-no-proxy="10.0.0.1"
+```
+
+`-node-http-proxy` & `-node-no-proxy` must only contain IP addresses and/or domain names.
+
+# Using a custom image registry
+
+Except for custom workload, the kubelet requires access to the "pause" container.
```

Comment:
custom workload?

Reply:
Own pods you deploy into the cluster. 
I wanted to somehow express that this registry does not modify your custom pods/deployments/etc. you still need to change your manifests to point to the registry.